package.json

```diff
@@ -20,7 +23,9 @@
     "babel-cli": "^6.26.0",
     "babel-jest": "^21.2.0",
     "babel-preset-env": "^1.6.1",
+    "@dschmidt/eventsourcemock": "1.1.0",
```

Comment:
why not `@dcos`?

Reply:
It's only temporal & @dcos would have been too much overhead